core/templates/pages/exploration-editor-page/modal-templates/state-version-history-modal.component.html

```diff
@@ -91,4 +94,10 @@ <h3>
     height: 4rem;
     width: 4rem;
   }
+  .error-message {
+    color: red;
```

Comment:
Please prefer hex value for colors, per [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).

Reply:
Done.